pl_bolts/datamodules/vocdetection_datamodule.py

```diff
@@ -152,16 +152,20 @@ def prepare_data(self) -> None:
         VOCDetection(self.data_dir, year=self.year, image_set="val", download=True)
 
     def train_dataloader(
-        self, batch_size: int = 1, image_transforms: Union[List[Callable], Callable] = None
+        self,
+        batch_size: int = 1,
+        transforms: Optional[List[Callable]] = [],
```

Comment:
I think we don't need `Optional` here if we set `[]` as the default value.
```suggestion
        transforms: List[Callable] = [],
```

Reply:
> We recently fixed some of the tests, so would you mind merging master branch? Also, it seems the change here is causing errors in CI, so would you mind resolving that, too?

I fixed the datamodule, but what do you mean by merging master branch? I believe this is based on the head of the master branch.

Reply:
I'm resolving this conversation but let me know if there's still something I need to do.